src/test-helpers.js

```diff
@@ -10,6 +10,16 @@ export function getRenderOutput(element) {
   return renderer.getRenderOutput();
 }
 
+class Wrapper extends Component {
+  render() {
+    return this.props.children;
+  }
+}
+
+export function renderFcIntoDocument(element) {
```

Comment:
`@Radium` (at least on class components) now exports with a HoC to inject context which is written as a function component, and `TestUtils.renderIntoDocument` requires a class component